### ISSUE TITLE
Questions about portal mode

### ISSUE THREAD

**Issue by sjsorock** [coldenate/rollover-daily-todos#6]
In portal mode, all of my notes for the day are referenced, not just TODO's

**Comment by coldenate** [comment]
Hi! Although I aim to improve the Portal mode's specificity, as of right now it is just a demo showing and testing the convenience of portals. RemNote's default behavior of portals causes some non-todos to be portaled in as well due to the parent rems being shown. 
This will go away in the future if the RemNote SDK implements a way for plugins to specify exactly what is shown and hidden in a portal. (I imagine white/blacklist style.)

So in summary, it's planned to not do what you are seeing. Ideally, it is supposed to be just Todos, but there are some current limitations.

**Comment by coldenate** [comment]
Closing this for now, feel free to comment a followup here tho
